airbyte-integrations/connectors/destination-s3/src/main/resources/spec.json

```
@@ -145,7 +146,7 @@
                     }
                   },
                   {
-                    "title": "xz",
+                    "title": "XZ",
```

```suggestion
                    "title": "xz",
```
These types are always in lower case, I don't think it is necessary to put them in upper case.